advanced/spec/hashTable_test.js

```diff
@@ -0,0 +1,11 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import HashTable from '../src/hashTable.js'
+
+describe('HashTable', () => {
+
+  it('exists', () => {
+    expect(HashTable).to.be.a('function')
+  })
+
```

Comment:
I have been testing in a repl by function because I am being a bad girl this week lol. Have been working on tests since lunch but they are not done. I was having trouble conceptualizing the structure of the data structure itself so I was unsure how to structure my tests. I feel confident on how to write my tests now and am currently working on it.